hazelcast/src/test/java/com/hazelcast/map/MapSetTTLBackupTest.java

```diff
@@ -36,51 +35,51 @@
 import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
+import org.junit.runners.Parameterized.Parameters;
+import org.junit.runners.Parameterized.UseParametersRunnerFactory;
 
 import java.util.concurrent.TimeUnit;
 
 import static com.hazelcast.test.backup.TestBackupUtils.assertBackupEntryEqualsEventually;
 import static com.hazelcast.test.backup.TestBackupUtils.assertBackupEntryNullEventually;
+import static com.hazelcast.test.backup.TestBackupUtils.newMapAccessor;
 
-@Category({QuickTest.class, ParallelTest.class})
 @RunWith(Parameterized.class)
-
-@Parameterized.UseParametersRunnerFactory(HazelcastSerialParametersRunnerFactory.class)
+@UseParametersRunnerFactory(HazelcastParallelParametersRunnerFactory.class)
+@Category({QuickTest.class, ParallelTest.class})
 public class MapSetTTLBackupTest extends HazelcastTestSupport {
 
-    private TestHazelcastInstanceFactory factory;
-    private final int NINSTANCE = 3;
-    private HazelcastInstance[] instances;
+    private static final int CLUSTER_SIZE = 3;
 
-    @Parameterized.Parameters(name = "inMemoryFormat: {0}")
+    @Parameters(name = "inMemoryFormat:{0}")
     public static Object[] memoryFormat() {
-        return new Object[] {InMemoryFormat.BINARY, InMemoryFormat.OBJECT};
+        return new Object[]{
+                InMemoryFormat.BINARY,
+                InMemoryFormat.OBJECT,
+        };
     }
 
-    @Parameterized.Parameter
+    @Parameter
     public InMemoryFormat inMemoryFormat;
 
     @Rule
     public RuntimeAvailableProcessorsRule runtimeAvailableProcessorsRule = new RuntimeAvailableProcessorsRule(4);
 
+    protected HazelcastInstance[] instances;
```

Comment:
I need access to this in a future EE test, to change the order of the created instances.